src/CMakeLists.txt

```diff
@@ -12,7 +12,10 @@ set(INEXOR_SOURCE_FILES
     vulkan-renderer/settings_decision_maker.cpp
     vulkan-renderer/time_step.cpp
 
-	vulkan-renderer/input/keyboard_mouse_data.cpp
+    vulkan-renderer/exceptions/exception.cpp
```

Comment:
```suggestion
```

Not required.

Reply:
I deleted `exceptions.cpp` and removed this entry.

Reply:
Done.